.github/workflows/deploy-docker-images.yml

```diff
@@ -4,78 +4,52 @@ name: Push docker images to GitHub container registry
 
 on:
   push:
```

Comment:
It should also run on `tags` to have tagged images.

Reply:
Now it will build on everything: tags, branches, etc. I don't think we need to restruct it to particular tags and branches.

Reply:
Ah, I see!